default/scripting/species/SP_KILANDOW.focs.txt

```diff
@@ -16,8 +16,36 @@ Species
     ]
 
     defaultfocus = "FOCUS_RESEARCH"
-    likes = ["FOCUS_RESEARCH" "FRACTAL_GEODES_SPECIAL" "MIMETIC_ALLOY_SPECIAL" "CRYSTALS_SPECIAL" "ECCENTRIC_ORBIT_SPECIAL" "PHILOSOPHER_SPECIAL" "TIDAL_LOCK_SPECIAL" "PLC_EXPLORATION" "PLC_LIBERTY" "PLC_NATIVE_APPROPRIATION" "PLC_TECHNOCRACY" ]
-    dislikes = ["BLD_SHIPYARD_CON_NANOROBO" "BLD_SHIPYARD_CON_GEOINT" "BLD_SHIPYARD_ORG_ORB_INC" "BLD_CLONING_CENTER" "BLD_SPACE_ELEVATOR" "BLD_SCANNING_FACILITY" "ELERIUM_SPECIAL" "RESONANT_MOON_SPECIAL" "WORLDTREE_SPECIAL" "MONOPOLE_SPECIAL" "PLC_BUREAUCRACY" "PLC_CHECKPOINTS" "PLC_MARTIAL_LAW" "PLC_DIVINE_AUTHORITY" ]
+    likes = [
+        "FOCUS_RESEARCH"
+        "FRACTAL_GEODES_SPECIAL"
+        "MIMETIC_ALLOY_SPECIAL"
+        "CRYSTALS_SPECIAL"
+        "ECCENTRIC_ORBIT_SPECIAL"
+        "PHILOSOPHER_SPECIAL"
+        "TIDAL_LOCK_SPECIAL"
+        "PLC_EXPLORATION"
+        "PLC_LIBERTY"
+        "PLC_NATIVE_APPROPRIATION"
+        "PLC_TECHNOCRACY"
+    ]
+    dislikes = [
+        //"BLD_CLONING_CENTER"       // disabled content
+        "BLD_CLONING_CENTER"
```

Comment:
ahhhh

Reply:
#%&(§§.+§$gpffffff yes i get it why reviewing is necessary .. doesnt meen i have to like it tho =)

Reply:
done fixed it up with previous commit so this should be outdated now.